src/rsa/padding.rs

```diff
@@ -164,9 +164,79 @@ pub struct PSS {
     pub digest_alg: &'static digest::Algorithm,
 }
 
-// Maxmimum supported output length for PSS padding.
+#[cfg(feature = "rsa_signing")]
+// Maximum supported length of the salt in bytes.
+// In practice, this is constrained by the maximum digest length.
+const MAX_SALT_LEN: usize = 512 / 8;
```

Comment:
I don't think we need this. We can just use `digest::MAX_OUTPUT_LEN` where necessary.